library/general/src/lib/installation/proposal_client.rb

```diff
@@ -132,6 +137,11 @@ def run
     #     The caller will format each item as a HTML list item (`<li>`). The
     #     proposal can contain hyperlinks with IDs listed in the list `links`.
     #
+    #   * **`"label_proposal"`** [Array<String>, nil]
```

Comment:
raw_proposal does *not* have the right format: it can contain rich text hyperlinks (eg. Bootloader: `<font color="red">Do not install...`)

I'll ask around for a  name that  people would consider fitting. BTW, "shrinked" is gram-not-matical: http://www.thefreedictionary.com/shrink